src/App/Exercises/Js/HitTheMoleGame/styles.css

```diff
@@ -0,0 +1,93 @@
+.molegame {
+  font-family: 'Poppins';
+  font-style: normal;
+}
+
+.molegame h2 {
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 34px;
+}
+
+.molegame-start-content {
+  margin: 3% 0 0 4%;
+}
+
+.molegame-start-result {
+  text-align: center;
+  font-weight: 500;
+  font-size: 28px;
+  line-height: 48px;
+}
+
+.molegame-start-settings {
+  display: flex;
+  flex-wrap: nowrap;
+}
+
+.molegame-start-settings p {
+  flex-basis: 20%;
+  height: 34px;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 34px;
+}
+
+.molegame-start-settings button {
+  width: 163px;
+  height: 56px;
+  background: #23213d;
+  border: none;
+  border-radius: 8px;
+  margin: 0 2%;
+  color: #ffffff;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 31px;
+  cursor: pointer;
+}
+
+.game-btn-active {
+  background: #fe3756 !important;
+}
+
+.molegame-setting {
+  margin: 0 2%;
+  width: 163px;
+  height: 56px;
+  background: #ebebeb;
+  border: none;
+  border-radius: 8px;
+  font-weight: 500;
+  font-size: 22px;
+  line-height: 31px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.fields-container {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 20%;
+}
+
+.mole-field {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: #ebebeb;
+  border-radius: 8px;
+  width: 103.77px;
+  height: 103.77px;
+  margin: 2%;
+  cursor: pointer;
+}
+
+.correct-field {
+  background: #e2fbd7;
+}
+
+.correct-field {
```

Comment:
Tutaj masz złą nazwę klasy. Jak tutaj ustawiłem `wrong-field` to jest poprawa, ale jeszcze nie jest idealnie 😄

Reply:
Ugrzęzłam i nie wiem w sumie jak to dalej usprawnić, bo jak klikne na puste pole to na wszystkich pustych polach robi mi się czerwono

Reply:
Zwróć uwagę na to, że jak masz `wrong click` to ustawiasz jedną flagę i na podstawie tej jednej flagi ustawiasz czerwone pole - czyli wszystkie "złe pola". Tutaj dla każdego pola osobno trzeba ten stan trzymać. Widzę dwa sposoby na trzymanie tego: to, co my pisaliśmy w czasie zajęć, abo ta flaga, gdzie trzymasz wrong click nie powinna być flagą. Powinna być IDkiem (indexem) pola, które jest `wrong`.

Reply:
Teraz mi wszystko działa jak powinno

Reply:
Potwierdzam, jest ok 😃  I ciekawy pomysł na rozwiazanie, działa idealnie jak widać, lepiej niż nasze rozwiązanie z zajęć 😄 good job 👍 💪